### PR TITLE
framework/wifi_utils: Fix bad log in scan function

### PR DESCRIPTION
**Pull request by SeonghoByeon**
- Modify a bad log in wifi_utils_scan_ap

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [4993cd361b7ed6ede151f129d7726e1fd25e49df] - Code Rule Check OK.

File: framework/src/wifi_manager/wpa_wifi_utils.c [hunk 1]
```diff
@@ -314,13 +314,12 @@ wifi_utils_result_e wifi_utils_scan_ap(void *arg)
...
-	ndbg("[WU] Register Scan Callback(%d)\n", ret);
 	ret = WiFiScanNetwork();
-	if (ret == SLSI_STATUS_SUCCESS) {
-		ndbg("[WU] WiFi scan success(%d)\n", ret);
-		wuret = WIFI_UTILS_SUCCESS;
+	if (ret != SLSI_STATUS_SUCCESS) {
+		ndbg("[WU] [ERR] WiFi scan fail(%d)\n", ret);
+		return wuret;
 	} 
-	ndbg("[WU] WiFi scan fail(%d)\n", ret);
```
Comment (davidfather) [inline framework/src/wifi_manager/wpa_wifi_utils.c]: Success debug message is missing here.

Comment (SeonghoByeon) [reply framework/src/wifi_manager/wpa_wifi_utils.c]: Successfully added the log for success. Thanks.

**Comment by seinfra** [comment]
Target : [94ca8d1975f00a6304e7f9c9d6f70d56cca56a5b] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [94ca8d1975f00a6304e7f9c9d6f70d56cca56a5b] - Code Rule Check OK.

**Pull request merged**